packages/atomic/src/components/search/result-template-components/atomic-result-image/atomic-result-image.tsx

```diff
@@ -36,27 +41,50 @@ export class AtomicResultImage implements InitializableComponent {
     return Array.isArray(value) ? value[0] : value;
   }
 
+  private handleImageError() {
+    let message =
+      'The image url is not valid. You might want to add a "fallback" property.';
+
+    if (this.fallback) {
+      message = 'The image url is not valid or could not be loaded.';
+      this.host.querySelector('img')!.src = this.fallback;
+    }
+    this.bindings.engine.logger.error(message, this.host);
```

Comment:
Could this be a warning if fallback is present and error if not?

Reply:
yes good idea, I did not know about the warning function 😁